extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessState.scala

```diff
@@ -121,6 +121,10 @@ class CustomStateStatus(val name: StatusName) extends StateStatus
   * @see overriding state definitions in [[OverridingProcessStateDefinitionManager]]
   */
 case class StateDefinitionDetails(displayableName: String,
-                                  icon: Option[URI],
+                                  icon: URI,
                                   tooltip: Option[String],
                                   description: Option[String])
```

Comment:
Maybe it's also time to change the tooltip and the description as well?

Reply:
Sounds good to me. Let me extract this change to another PR.